recipes/aws-c-event-stream/all/conanfile.py

```diff
@@ -1,7 +1,8 @@
-from conans import CMake, ConanFile, tools
 import os
```

Comment:
Please keep this order.

Reply:
Is a formatter doing this automatically?

Reply:
No it is just that I wrote theses recipes a long time ago here: #6346